src/core/qgsogcutils.h

```diff
@@ -565,6 +565,41 @@ class QgsOgcUtilsSQLStatementToFilter
                          QString &srsName,
                          bool &axisInversion );
 };
+
+/**
+ * \ingroup core
+ * \brief Utilities related to OGC CRS encodings.
+ * \note not available in Python bindings
+ */
+class CORE_EXPORT QgsOgcCRSUtils
```

Comment:
I think that it should be (QT-style)
```suggestion
class CORE_EXPORT QgsOgcCrsUtils
```